originatorToBeneficiary.go

```diff
@@ -56,7 +56,7 @@ func (ob *OriginatorToBeneficiary) Parse(record string) error {
 	ob.LineOne = value
 	length += read
 
-	if len(ob.LineOne) >= 35 {
+	if len(ob.LineOne) >= 35 && record[length-1:length] != "*" {
```

Comment:
Are there other records with this same bug?

Reply:
cc @mfdeveloper508

Reply:
the code is correct
`value, read, err := ob.parseVariableStringField(record[length:], 35)`
parseVariableStringField will return value and read 
if LineOne's size is 35 AND next position is delimiter, value is raw string (length is 35) and read is 36 (raw string + delimiter)

`length += (strings.Index(record[length:], "*") + 1)`
inputted LineOne can be great than max size, so the code tried to take last position of line one 

`if len(ob.LineOne) >= 35 && record[length-1:length] != "*" {`
the code will skip error case like as above  (lineone is max length string and next position is delimiter)

Reply:
Will we need to make a similar fix in other records? I was hoping the fix would only be within `parseVariableStringField`.

Reply:
to be honest I think that we need to parse element by delimiter and max length
It seems that the approach has a lot of side effects 
fixed same cases in other files

Reply:
Can we change parseVariableStringField to parse by delimiter and max length?

Reply:
okay, I will change this function to has delimiter and max length checker